manifests/dcgm/telegraf.pp

```diff
@@ -22,108 +28,108 @@
   } else {
     $ensure_parm = 'absent'
   }
-
-  #
+  
+ 
   # Setup nvidia-dcgm systemd service
   #
```

Comment:
I'd move this comment to be inside the `if $facts['nvdebugging'] != true` block. Reason is that the comment divides up the three main components when you are doing the install/setup.

Reply:
This is corrected.